plugin.video.tv3.cat/resources/lib/utils/Utils.py

```diff
@@ -1,97 +1,91 @@
-# -*- coding: utf-8 -*-
-
-'''
- TV3cat Kodi addon
- @author: jqandreu
- @contact: jqsandreu@gmail.com
-'''
-
-import urllib
-import urllib2
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import urllib.request, urllib.parse, urllib.error
```

Comment:
not py2compatible

Reply:
Also there are imports form future script